Resources/public/mapbender.element.digitizer.js

```diff
@@ -2086,6 +2098,12 @@
                 fontSize:    15
             });
 
+            var copyStyleData = getValueOrDefault(schema, 'copy.style', null);
```

Comment:
In my opinion the default value should be `false` not `null`

Reply:
`copy.style` is tree configuration node this should has own nodes if defined.
I prefer to let it be `null` or `undefined` but not `false`

Reply:
@atd-schubert @werrolf Hi, Merging is blocked - could you check the code and merge the changes?

Reply:
OK, then I misunderstood the value...